magentoerpconnect_transaction_id/models/sale.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2015 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp.addons.magentoerpconnect import sale
+from openerp.addons.connector.unit.mapper import mapping
+from openerp.addons.magentoerpconnect.backend import magento
+
+
+@magento(replacing=sale.SaleOrderImportMapper)
```

Comment:
@guewen Thank you for the link. I'll introduce a new hook `SalePaymentImportMapper` to avoid to add a new dependency on `base_transaction_id`.